README.md

```diff
@@ -275,13 +275,14 @@ use {
   commit = string,             -- Specifies a git commit to use
   lock = boolean,              -- Skip this plugin in updates/syncs
   run = string or function,    -- Post-update/install hook. See "update/install hooks".
-  requires = string or list -- Specifies plugin dependencies. See "dependencies".
+  requires = string or list    -- Specifies plugin dependencies. See "dependencies".
   config = string or function, -- Specifies code to run after this plugin is loaded.
   -- The following keys all imply lazy-loading
   cmd = string or list,        -- Specifies commands which load this plugin.
   ft = string or list,         -- Specifies filetypes which load this plugin.
   keys = string or list,       -- Specifies maps which load this plugin. See "Keybindings".
   event = string or list,      -- Specifies autocommand events which load this plugin.
+  func = string or list        -- Specifies functions which load this plugin.
```

Comment:
Nitpick: What if we call this keyword `fn` instead? It's slightly more concise, and common to some programming languages (e.g. Rust). This latter property might be true of `func` too, but I'm not sure.

Reply:
Sure I can change that. I have no hard feelings about it as I stated in the PR summary.
Asap I'm sitting at a PC again I'll change it.

Reply:
Done :heavy_check_mark: